app/src/main/java/com/mobiledevelopmentfoodapp/dto/Order.kt

```diff
@@ -4,7 +4,13 @@ package com.mobiledevelopmentfoodapp.dto
 //import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
-data class Order(var customerId: Int, var name: String, var products: ArrayList<Food>, var totalPrice: Double, @SerializedName("id") var orderId:Int = 0 ) {
+data class Order(
```

Comment:
Another data class revision

Reply:
Duplicates a previous review.  When that happens, first in wins.